mkwheelhouse.py

```diff
@@ -93,7 +94,11 @@ def build_wheels(packages, index_url):
         '--wheel-dir', temp_dir,
         '--find-links', index_url
     ]
-    args += packages
+    # Use a requirements.txt file -r/--requirements
+    if requirements is not None:
+        args += ['-r', requirements]
+    else:
+        args += packages
```

Comment:
This seems surprising to me: specifying a requirements file will cause any additionally specified packages to be silently ignored. We should either warn the user that they can't specify both `args.requirement` and `args.package` or always append any directly-specified packages.